### ISSUE TITLE
Epic: Product media management on Imajin (Debbie outerwear catalog, first tenant)

### ISSUE THREAD

**Issue by imajin-jin** [ima-jin/imajin-ai#1549]
## Product media management on Imajin

**First tenant: Debbie's outerwear catalog** (~800 SKUs, humans in winter jackets / outerwear, body shots). The concrete need is sorting product images by orientation (left / right / front / 3-4), which Claude 4.8 does well on everything *except* left/right — a known structural VLM weakness (mirror-flip augmentation destroys egocentric orientation). But the real target is bigger: **Debbie manages product on the platform**, where structured attributes (SKU, gender, colour, orientation) ride *with* each signed asset as queryable data, not trapped in folder names.

## The shape (the spine)

```
declared attribute vocab  →  classifier PROPOSES orientation  →  human CORRECTS on canvas (signed)  →  corrections RETRAIN
```

A caller (Debbie's ClaudeCode) walks her local product tree, parses SKU/gender/colour from the path and orientation from the filename convention, and uploads each image *with* its metadata `onBehalfOf` Debbie. The pose classifier fills in / verifies orientation. A browser shows image + attributes side-by-side; Debbie's overrides are logged and attributed — **the correction IS the consent event** (chat/model proposes, canvas is authoritative + signed, approving on the canvas signs). Those signed corrections become the ground-truth training set. The product tool and the labeling pipeline are the same surface.

## Decisions (Ryan, 2026-08-01)

- **Declared-vocab-lite**, not free-form jsonb: attributes have a declared schema (name → allowed values) so the UI renders toggles and the classifier has a closed label set.
- **All in `imajin-ai`** — this is core media/ml, not a vertical.
- **This first pass = the image classifier only** (the do-now child below). The metadata field, the correction browser, and the retrain loop are the future shape — stubbed here so the architecture is on record. **The ultimate product app will implement and extend these.**

## Children (dependency order)

1. **[do now] pose-orientation classifier service** — the one broken axis, solvable deterministically. Independent of the platform work; validated by a spike against Debbie's sorted samples.
2. **[future] structured `metadata` on assets** (declared-vocab-lite) — the foundation the rest sits on.
3. **[future] product metadata browser** — image + attribute panel, override-is-signed. Extends the media viewer (#1445/#1543).
4. **[future] retrain-from-corrections loop** — attested overrides → labeled training data.

## Relatives

- **#651** (`feat(ml): CLIP image classification endpoint`) — sibling classifier home; the orientation service folds into this endpoint family.
- **#1445 / #1543** — media viewer UI the correction browser builds on (does not duplicate).

## Folded in

**From #651 — feat(ml): CLIP image classification endpoint:**
- CLIP image classification is the ML service endpoint for the product-media pipeline. It provides real confidence + tags for .fair attribution on media assets. This is the classifier layer of the product-media epic, not a separate ML epic.

**From #1551 — feat(media): structured metadata on assets:**
- Structured metadata (declared-vocab-lite) is the foundation for product media management: writable on upload + PATCH, queryable, with closed vocab per owner/scope. This is the data layer of the product-media epic.

**From #1552 — feat(media): product metadata browser:**
- The product metadata browser (image + attribute panel, override-is-signed) is the UI layer of the product-media pipeline. It extends the media viewer with declared-vocab metadata editing. Each correction is a signed attestation — the consent event. This is the interaction layer of the product-media epic.

**From #1553 — feat(ml): retrain-from-corrections loop:**
- The retrain-from-corrections loop closes the product-media pipeline: signed, attributed corrections from the browser become human-labeled ground truth for classifier improvement. This is the feedback layer of the product-media epic.
